modules/uhttp/http.go

```
@@ -152,6 +152,11 @@ func (m *Module) Start() error {
 		m.lock.RLock()
 		listener := m.listener
 		m.lock.RUnlock()
+		if listener == nil {
+			m.log.Error("listener is nil, module was stopped before server is able to start")
```

nit: I would not log here, I would return an error.